### PR TITLE
Enhancement/highlights

### PR DESCRIPTION
**Pull request by jonnybazookatone**
Thought I'd push this first approach for issue #65, that is a simple access to highlights, as it's been collecting dust on my laptop. Works like so:

```python
>>> import ads
>>> q = ads.SearchQuery(q='star', hl=['abstract', 'body', 'title'])
>>> p = list(q)
>>> q.highlights(p[0])
{'abstract': [' in the early universe or in the ultra-dense core of neutron <em>stars.</em> The thermal radiation from the quarks']}
```

`hl` can take `['abstract', 'body', 'title']`, which are the available backend options at the ADS.

### CODE REVIEW

File: ads/search.py [hunk 1]
```diff
@@ -381,10 +384,18 @@ def __init__(self, query_dict=None, q=None, fq=None, fl=DEFAULT_FIELDS,
                 # cursors require unique field in the sort
                 if "id" not in sort and start is None:
                     sort = "{},id desc".format(sort)
+            if hl is not None:
+                hl_fl = [i for i in set(hl) if i in ['abstract', 'title', 'body']]
+                hl = "true"
```
Comment (vsudilov) [inline ads/search.py]: Does `True` work?

File: ads/search.py [hunk 2]
```diff
@@ -381,10 +384,18 @@ def __init__(self, query_dict=None, q=None, fq=None, fl=DEFAULT_FIELDS,
                 # cursors require unique field in the sort
                 if "id" not in sort and start is None:
                     sort = "{},id desc".format(sort)
+            if hl is not None:
+                hl_fl = [i for i in set(hl) if i in ['abstract', 'title', 'body']]
```
Comment (vsudilov) [inline ads/search.py]: Consider throwing a warning or exception if a non-supported value for `hl` is given -- I don't feel strongly one way or another personally.

File: ads/search.py (hunk 1 above)
Comment (jonnybazookatone) [inline ads/search.py]: It's capitalising it later on the formatting of the URL, and so Solr borks.

**Comment by vsudilov** [comment]
Cool thanks

**Pull request merged**